opencti-platform/opencti-graphql/src/modules/entitySetting/entitySetting.ts

```diff
@@ -93,6 +93,20 @@ export const ENTITY_SETTING_DEFINITION: ModuleDefinition<StoreEntityEntitySettin
     { name: 'platform_hidden_type', label: 'Platform hidden type', type: 'boolean', mandatoryType: 'external', editDefault: true, multiple: false, upsert: false, isFilterable: false },
     { name: 'enforce_reference', label: 'Enforce reference', type: 'boolean', mandatoryType: 'external', editDefault: true, multiple: false, upsert: false, isFilterable: false },
     { name: 'attributes_configuration', label: 'Attributes configuration', type: 'string', format: 'json', mandatoryType: 'no', editDefault: false, multiple: false, upsert: false, schemaDef: attributeConfiguration, isFilterable: false },
+    { name: 'overview_layout_customization',
+      label: 'Overview layout customization',
+      type: 'object',
+      format: 'standard',
+      mandatoryType: 'no',
+      editDefault: false,
+      multiple: true,
+      upsert: true,
+      isFilterable: false,
+      mappings: [
+        { name: 'key', label: 'Key', type: 'string', format: 'short', mandatoryType: 'internal', upsert: true, editDefault: false, multiple: false, isFilterable: false },
+        { name: 'order', label: 'Order', type: 'numeric', precision: 'integer', mandatoryType: 'internal', upsert: true, editDefault: false, multiple: false, isFilterable: false },
```

Comment:
I think we can completely get rid of the "order" everywhere.
We store an array ; the order is the index in the array.
When sending the new array, we send the array in the right order, no need for sorting it manually before storing in DB. This way we have a consistent API.

It's up to the frontend to send the right payload, the right array in the right order.
Today it can use a form with a numeric input for the order ; tomorrow it will be a drag'n'drop.... it's really a frontend problematic.